### ISSUE TITLE
Add "tab" support

### ISSUE THREAD

**Issue by romaninsh** [atk4/ui#176]
Add tab support like https://semantic-ui.com/modules/tab.html#/third/c

 - static content
 - loadable (through call-back)
 - (maybe)automatically add #/ path somehow
 - make sure javascripts are executed
 - define if we want to cache dynamic content.

``` php
$tabs = $this->add('Tabs');
$tabs->addTab('Hello')->add('HelloWorld');
$tabs->addTab('Dynamic', function($tab) {
    $tab->add('LoremIpsum');
});
```

**Comment by DarkSide666** [comment]
sounds good
definitely this is _must-have_ in UI

**Comment by DarkSide666** [comment]
initial implementation is here https://github.com/atk4/ui/commit/abb1965b2a3a335319b12d0ff15ef4bf83715e7e

**Comment by romaninsh** [comment]
was fully implemented and released in 1.3
